index.html

```diff
@@ -1,236 +1,200 @@
-          </div>
-          
-          <p class="section-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-          
-          <ul class="section-skills">
-            <li class="skills">html</li>
-            <li class="skills">css</li>
-            <li class="skills">javascript</li>
-          </ul>
-          
-          <a href="#" class="btn-text">See Project</a>
+                </ul>
+            </section>
         </div>
-    </section>
-
-      <section class="section-label desk-content">
-        <img class="portfolios" src="Images/Snapshoot2.png" alt="Portfolio1">
-        <img class="desk-img" src="Images/Desktop2.png" alt="desk-img">
-        <div class="label-inside">
-
-        <h2 class="section-title">Multi-Post <br class="line-break"> Stories  </h2>
-        <div class="section-options">
-          <a href="#" class="optionBold">canopy</a>
-          <span class="dot"></span>
-          <a href="#" class="options">Back End Dev</a>
-          <span class="dot"></span>
-          <a href="#" class="options">2015</a>
+        <div class="grid-container">
+            <div class="grid-item">
+                <div class="grid-item-photo" id="photo1" data-modal-target="#modal1">
+                </div>
+                <div class="grid-item-section">
+                    <h2 class="title">Tonic</h2>
+                    <ul class="sub-title">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>
+                        A daily selection of privately personalized reads; no account or sign-ups required.</p>
+                    <ul class="category">
+                        <li><a href="index.html">html</a></li>
+                        <li><a href="index.html">css</a></li>
+                        <li><a href="index.html">javaScript</a></li>
+                    </ul>
+                    <button type="button" class="project" data-modal-target="#modal1">See Project</button>
+                </div>
+            </div>
+            <div class="grid-item">
+                <div class="grid-item-photo" id="photo2" data-modal-target="#modal0">
+                </div>
+                <section class="grid-item-section">
+                    <h2 class="title">Multi-Post Stories</h2>
+                    <ul class="sub-title">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>
+                        A daily selection of privately personalized reads; no account or sign-ups required.</p>
+                    <ul class="category">
+                        <li class="category-list"><a href="index.html">html</a></li>
+                        <li class="category-list"><a href="index.html">css</a></li>
```

Comment:
- [ ] As per the project requirements kindly make sure the projects are generated dynamically with javascripts. :recycle: